sample-conf.yaml

```diff
@@ -83,7 +83,25 @@ services:
         "valid_until": "2020-01-01"
 
     # The LSAT value in satoshis for the service.
-    price: 1     
+    price: 1
+
+    # Whether or not a gRPC server is available to query price data from. If
+    # this option is set then the 'price' option is ignored.
+    usepricerserver: true
+
+    # Options to use for connection to the price serving gRPC server. If the
+    # 'usepricerserver' option is set then these config options must be set.
+    pricerconfig:
```

Comment:
What do you think about the following hierarchy of options instead?
```yaml

price: 0
dynamicprice:
    enabled: true
    grpcaddress: "127.0.0.1:10010"
    insecure: false
    tlscertpath: "path-to-pricer-server-tls-cert/tls.cert"
```

That way we need an additional `nil` check for the `DynamicPrice` field but only introduce one new top-level field.
Just a thought, might be a bit easier to understand that way?